### ISSUE TITLE
Ubuntu's apt method cannot install openvino2022.2.0

### ISSUE THREAD

**Issue by 982118809** [openvinotoolkit/openvino#13277]
Hello，I want to install openvino2022.2.0 through apt, but  `https://apt.repos.intel.com/openvino/2022 bionic main` does not have this version.
    Ubuntu: 18.04
**Code:**
```
wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB 

apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB 

echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" >> /etc/apt/sources.list.d/intel-openvino-2022.list

apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB 

apt update 
```
Log:
```
--2022-09-29 01:58:20--  https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
Resolving apt.repos.intel.com (apt.repos.intel.com)... 104.127.181.69, 2600:1408:c400:392::4b23, 2600:1408:c400:390::4b23
Connecting to apt.repos.intel.com (apt.repos.intel.com)|104.127.181.69|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 3796 (3.7K) [binary/octet-stream]
Saving to: ‘GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB’

GPG-PUB-KEY-INTEL-S 100%[===================>]   3.71K  --.-KB/s    in 0s      

2022-09-29 01:58:20 (437 MB/s) - ‘GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB’ saved [3796/3796]

OK
OK
Get:1 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/ InRelease [3,626 B]
Get:2 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Hit:3 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic InRelease
Hit:4 http://archive.ubuntu.com/ubuntu bionic InRelease
Get:5 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Ign:6 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease
Hit:7 http://ppa.launchpad.net/cran/libgit2/ubuntu bionic InRelease
Get:8 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease [1,581 B]
Get:9 https://apt.repos.intel.com/openvino/2022 bionic InRelease [5,532 B]
Hit:10 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release
Hit:11 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic InRelease
Get:12 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [83.3 kB]
Hit:13 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease
Get:14 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Packages [913 kB]
Get:16 https://apt.repos.intel.com/openvino/2022 bionic/main amd64 Packages [2,828 B]
Get:17 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [2,990 kB]
Get:18 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [2,322 kB]
Get:19 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [1,545 kB]
Get:20 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [3,422 kB]
Fetched 11.5 MB in 4s (2,978 kB/s)
Reading package lists... Done
Building dependency tree       
Reading state information... Done
20 packages can be upgraded. Run 'apt list --upgradable' to see them.
```
**check OpenVINO version**
`apt-cache search openvino `
Log:
```
libopenvino-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit Runtime
libopenvino-dev-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit Developer Tools
openvino-libraries-dev-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit Libraries and Developer tools
openvino-opencv - OpenCV for OpenVINO(TM) toolkit
libopenvino-auto-plugin-2022.1.0 - Inference Engine Auto plugin
libopenvino-intel-gpu-plugin-2022.1.0 - Intel® Processor Graphics
libopenvino-intel-vpu-plugin-2022.1.0 - Intel® Movidius™ VPU
openvino-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit
libopenvino-hetero-plugin-2022.1.0 - Inference Engine Runtime for Intel® Gaussian Neural Accelerator
libopenvino-intel-hddl-plugin-2022.1.0 - Intel® Vision Accelerator Design with Intel® Movidius VPUs
openvino-libraries-dev - Intel(R) Distribution of OpenVINO(TM) Toolkit Libraries and Developer tools
openvino-opencv-2022.1.0 - OpenCV for OpenVINO(TM) toolkit
libopenvino-auto-batch-plugin-2022.1.0 - Inference Engine Automatic Batching plugin
libopenvino-python-2022.1.0 - Inference Engine Python API
openvino-libraries - Intel(R) Distribution of OpenVINO(TM) Toolkit Libraries
openvino-samples-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit Samples
libopenvino-intel-cpu-plugin-2022.1.0 - Intel® CPU
libopenvino-intel-gna-plugin-2022.1.0 - Intel® Gaussian Neural Accelerator
openvino - Intel(R) Distribution of OpenVINO(TM) Toolkit
openvino-libraries-2022.1.0 - Intel(R) Distribution of OpenVINO(TM) Toolkit Libraries
```
It only has openvino2022.1.0.
Thanks

**Comment by ilya-lavrenov** [comment]
Hi @982118809 

As you can see from installation documentation https://docs.openvino.ai/2022.2/openvino_docs_install_guides_installing_openvino_linux_header.html, Debian packages are not available in 2022.2.
They will be returned back in 2022.3 in an updated form. 

For 2022.2 you can download archives from github https://github.com/openvinotoolkit/openvino/releases/tag/2022.2.0 and extract it to any location you want, the folders structure is the same as in debian packages. Sorry for inconvenience

**Comment by 982118809** [comment]
Okay, I understand now, thanks!
By the way, when do you plan to release the 2022.3 version.

**Comment by ilya-lavrenov** [comment]
> Okay, I understand now, thanks!
> By the way, when do you plan to release the 2022.3 version.

I suppose beginning of 2023

**Comment by avitial** [comment]
Closing. Feel free to reopen and ask any questions related to this topic.

**Comment by ilya-lavrenov** [comment]
OpenVINO debian packages are available since 2022.3 release 
https://docs.openvino.ai/2022.3/openvino_docs_install_guides_installing_openvino_apt.html

**Comment by SaraDanaKablTalabani** [comment]
Hi,
I use Ubuntu 24.04.2 LTS noble with NUC with Intel(R) Core(TM) Ultra 7 155H and i got this 
E: The repository 'https://apt.repos.intel.com/openvino/2024 ubuntu22 InRelease' is not signed. 
here 
sudo apt update
Hit:1 http://it.archive.ubuntu.com/ubuntu noble InRelease
Hit:2 http://it.archive.ubuntu.com/ubuntu noble-updates InRelease                               
Hit:3 http://it.archive.ubuntu.com/ubuntu noble-backports InRelease                             
Hit:4 https://repositories.intel.com/gpu/ubuntu noble InRelease                                 
Hit:5 http://security.ubuntu.com/ubuntu noble-security InRelease                                
Hit:6 https://download.docker.com/linux/ubuntu noble InRelease                                  
Get:7 https://apt.repos.intel.com/openvino/2024 ubuntu22 InRelease [7776 B]
Err:7 https://apt.repos.intel.com/openvino/2024 ubuntu22 InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY BAC6F0C353D04109
Reading package lists... Done
W: Target Packages (client/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-i386/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Translations (client/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11 (client/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11 (client/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons-small (client/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons (client/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons-hidpi (client/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target CNF (client/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target CNF (client/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Packages (client/binary-i386/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Packages (client/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Translations (client/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11 (client/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11 (client/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons-small (client/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons (client/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons-hidpi (client/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target CNF (client/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target CNF (client/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: GPG error: https://apt.repos.intel.com/openvino/2024 ubuntu22 InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY BAC6F0C353D04109
E: The repository 'https://apt.repos.intel.com/openvino/2024 ubuntu22 InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: Target Packages (client/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-i386/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Translations (client/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11 (client/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11 (client/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons-small (client/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons (client/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target DEP-11-icons-hidpi (client/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target CNF (client/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target CNF (client/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-gpu.list:1
W: Target Packages (client/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Packages (client/binary-i386/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Packages (client/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target Translations (client/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11 (client/dep11/Components-amd64.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11 (client/dep11/Components-all.yml) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons-small (client/dep11/icons-48x48.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons (client/dep11/icons-64x64.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target DEP-11-icons-hidpi (client/dep11/icons-64x64@2.tar) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target CNF (client/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
W: Target CNF (client/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/intel-gpu-noble.list:1 and /etc/apt/sources.list.d/intel-graphics.list:1
